### PR TITLE
Make section assigner scrollable

### PR DESCRIPTION
**Pull request by kobryan0619**
We have had 2 Zendesk tickets this week with teachers saying they couldn't assign content from the Multiple Section Pop up - in both cases, this was because they had so many sections that the button to "confirm" was hidden.  

What this does is make the "sections" portion scrollable so that the "confirm" button is always viewable. 

This is an example of a teacher with a lot of sections who cannot see the "confirm" button.
image

Here's what it looks like with this change:
image

This video will also show it in action: https://github.com/code-dot-org/code-dot-org/assets/24883357/1c0ae154-b169-4df8-8e5c-d63c0e7a1275

UPDATE: This video shows the accessibility with tab navigation:
https://github.com/code-dot-org/code-dot-org/assets/24883357/5d6ea002-6112-453d-9af9-21311f76fc74

## Links

- [Zendesk ticket #1](https://codeorg.zendesk.com/agent/tickets/458639)
- [TEACH-690 Make multiple section assigner pop-up scrollable](https://codedotorg.atlassian.net/browse/TEACH-690?atlOrigin=eyJpIjoiZDQ5Mzg4ZmI1YjY2NGRlOGI4NjI2ZGFmOWY5MGVmZDQiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by hannahbergam — APPROVED** [review]
I wish it were obvious it's a scroll without needing to hover, but I'm sure this is the way most of our scrollbars work. LGTM- love the simplicity! Also... do you know what the accessible experience is here tab-nav-wise? Not necessary for this ticket, but maybe a followup?

**Comment by kobryan0619** [comment]
@hannahbergam I made the changes to the PR to improve accessibility - SO MUCH BETTER!  Thanks for walking me through that!  I included an updated video in the description that shows how it works now.

File: apps/src/templates/MultipleSectionsAssigner.jsx [hunk 1]
```diff
@@ -202,7 +202,12 @@ const MultipleSectionsAssigner = ({
       </div>
       <div className={moduleStyle.sectionList}>
         <Heading5>{i18n.yourSectionsList()}</Heading5>
-        <div className={moduleStyle.sectionListOptionsContainer}>
+        <div
+          role="region"
+          aria-label="List of all teacher-created sections available to assign"
```
Comment (etaderhold) [inline apps/src/templates/MultipleSectionsAssigner.jsx]: Should this be a translated string, or do we make no particular guarantee that our accessibility tags will be understandable outside of the English language?

**Comment by hannahbergam** [comment]
> @hannahbergam I made the changes to the PR to improve accessibility - SO MUCH BETTER! Thanks for walking me through that! I included an updated video in the description that shows how it works now.

Omg HECK YES thank you for the follow up! It looks so great in that video!

File: apps/src/templates/MultipleSectionsAssigner.jsx (hunk 1 above)
Comment (hannahbergam) [reply apps/src/templates/MultipleSectionsAssigner.jsx]: I'm not sure what our protocol is on this, but it seems like a generally good idea to make it translatable!

**Review by hannahbergam — APPROVED** [review]
Yay thanks so much for doing this! I think it might be worth making this built in for all uses of the accessible dialog (think something like 'isScrollingEnabled' as a param for the dialog). Again, totally makes sense as a followup task in our 'up next'.

**Pull request merged**